DataManager.js

```diff
@@ -1,12 +1,45 @@
 var MongoClient = require('mongodb').MongoClient;
 var url = "mongodb://localhost:27017/soen341db";
 
-MongoClient.connect(url, function(err, db) {
-  if (err) {
-  	console.log("Error creating database");
-  	throw err;
-  } else {
-  	console.log("Database has successfully been created!");
-  }  
-  db.close();
+MongoClient.connect(url, function(error, db) {
+	if (error) {
+		console.log("Error creating database");
+		throw error;
+	} else {
+		console.log("Database has successfully been created!");
+	}  
+
+	var dbo = db.db("database");
+
+	// Creating a collection to store data about posts
+	dbo.createCollection("posts", function (error, response) {
+		if (error) {
+			console.log("Unable to create collection \"Posts\"");
+			throw error;
+		} else {
+			console.log("Collection \"Posts\" has successfully been created!");			
+		}
+		db.close();
+	});	
```

Comment:
I'm aware this is a prototype, but this comment is purely for optimization purpose (reducing the number of lines of code), I think MongoDB automatically creates a collection if it doesn't not exists. Therefore when (further down the code) when your adding an array of objects to the collection "posts", the "posts" collection will be created on the fly spot if it doesn't exist!